amti/clis/extract.py

```diff
@@ -4,29 +4,35 @@
 
 import click
 
-from amti import actions
+from amti import clis
 
 
 logger = logging.getLogger(__name__)
 
 
-@click.command(
+@click.group(
     context_settings={
         'help_option_names': ['--help', '-h']
     })
-@click.argument(
-    'batch_dir',
-    type=click.Path(exists=True, file_okay=False, dir_okay=True))
-@click.argument(
-    'output_dir',
-    type=click.Path(exists=True, file_okay=False, dir_okay=True))
-def extract_xml(batch_dir, output_dir):
-    """Extract XML data from assignments in BATCH_DIR to OUTPUT_DIR.
-
-    Given a directory (BATCH_DIR) that represents a batch of HITs that
-    have been reviewed and saved, extract the XML data from the
-    assignments to OUTPUT_DIR.
+def extract():
+    """Extract data from a batch to various formats.
+
+    See the subcommands for extracting batch data into a specific
+    format.
     """
-    actions.extract_xml(batch_dir=batch_dir, output_dir=output_dir)
+    pass
+
+
+subcommands = [
```

Comment:
If its easy, `tabular` can be replaced with the possible file formats. So usage will be like 
`amti extract xml / json / jsonl / tsv ...`

Reply:
Thanks Chandra, the reason I chose to have a separate tabular command is because the tabular format is fairly distinct from the XML format. `xml` returns a directory tree whereas `tabular` returns one flat file / table. Perhaps there's a better name we could come up with for the `xml` command that describes the structure of the extracted content rather than the returned format?

Reply:
My thinking was commands should map onto the structure of the extracted data, while a flag can set the serialization format.